corehq/apps/app_manager/tasks.py

```diff
@@ -0,0 +1,9 @@
+from celery.task import task
+from corehq.apps.users.models import CommCareUser
+
+
+@task
```

Comment:
Do you have a sense of what the priority is of this task?  By default this task will run on the main celery queue which is the high priority queue.  If you think this is a longer running / lower priority task then we could move it to the background_queue (see https://github.com/dimagi/commcare-hq/blob/9e7b34ca0cded7d8da8c89c1f081c7d5460741f6/corehq/apps/users/tasks.py#L31 for an example).

No need to block on this, just might be worth giving some thought.